mods/sethome/init.lua

```diff
@@ -94,9 +109,6 @@ minetest.register_chatcommand("sethome", {
 	func = function(name)
 		name = name or "" -- fallback to blank name if nil
 		local player = minetest.get_player_by_name(name)
-		if player and sethome.set(name, player:get_pos()) then
-			return true, S("Home set!")
-		end
-		return false, S("Player not found!")
+		minetest.show_formspec(player:get_player_name(), "sethome:sethomedialog", formspec)
```

Comment:
Are you doing this on purpose?
```suggestion
		minetest.show_formspec(name, "sethome:sethomedialog", formspec)
		return true
```

Reply:
Oops, my mistake. I left it there while I was trying to fix an error. Fixed!